extension/src/plots/paths/collect.test.ts

```diff
@@ -134,6 +139,46 @@ describe('collectPaths', () => {
     }
   })
 
+  it('should update the existing path types when the paths are recollected', () => {
```

Comment:
[N] could be more specific here as it is a regression test. E.g `should collect path types after an error is returned for a new path`.